metrics/eventLoopLag.js

```diff
@@ -32,43 +32,43 @@ module.exports = (meter, {prefix, labels, eventLoopMonitoringPrecision}) => {
 
   meter.createObservableGauge(prefix + NODEJS_EVENTLOOP_LAG_MIN, {
     description: 'The minimum recorded event loop delay.'
-  }, (observable) => {
+  }).addCallback(async (observable) => {
```

Comment:
Those callback functions don't need to use `async`.
It's optional according to the types: https://github.com/open-telemetry/opentelemetry-js/blob/d3adc22c4715d77e3a6913d0e14950fd68d470f9/experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts#L104

 Please remove that here again.